src/lib/party.ts

```diff
@@ -492,6 +503,52 @@ type UnfinishedSignature = undefined | LazySignature | string;
 
 type LazyControl = Control | LazySignature | LazyProof;
 
+class Token {
+  readonly id: string;
+  readonly parentTokenId: string;
+  readonly tokenOwner: PublicKey;
+
+  constructor(options: { tokenOwner: PublicKey; parentTokenId?: string }) {
+    let { tokenOwner, parentTokenId } = options ?? {};
+
+    // Reassign to default tokenId if undefined
+    parentTokenId ??= getDefaultTokenId();
+
+    // Check if we can deserialize the parentTokenId
+    try {
+      Ledger.fieldOfBase58(parentTokenId);
+    } catch (e) {
+      throw new Error(`Invalid parentTokenId\n(error: ${(<Error>e).message})`);
```

Comment:
Did you mean to have this error message? 

`Invalid parentTokenId\n(error: ${e.message})`;

If not, I'm just curious what this does `<Error>` tag is.

Reply:
Oh nice!